src/ishikawa_backend.erl

```diff
@@ -59,9 +63,9 @@ tcbcast(Message) ->
     gen_server:call(?MODULE, {tcbcast, Message}, infinity).
 
 %% Deliver a message.
--spec tcbdeliver(message(), timestamp()) -> ok.
-tcbdeliver(Message, Timestamp) ->
-    gen_server:call(?MODULE, {tcbdeliver, Message, Timestamp}, infinity).
+-spec tcbdeliver(actor(), message(), timestamp()) -> ok.
+tcbdeliver(Actor, Message, Timestamp) ->
```

Comment:
What do you mean exactly? This function is called by a node that received this function and after sending it to all neighbors and receiving acks.